torch/distributed/pipeline/sync/phony.py

```diff
@@ -19,9 +19,10 @@
 
 
 def get_phony(device: torch.device, *, requires_grad: bool) -> Tensor:
-    """Gets a phony. Phony is tensor without space. It is useful to make
-    arbitrary dependency in a autograd graph because it doesn't require any
-    gradient accumulation.
+    """
+    Get a phony. Phony is tensor without space.
+
+    It is useful to make arbitrary dependency in a autograd graph because it doesn't require any gradient accumulation.
```

Comment:
```suggestion
    """Get a phony, a tensor without space.
    
    It is useful to make arbitrary dependency in a autograd
    graph because it doesn't require any gradient accumulation.
```